pywhat/printer.py

```diff
@@ -98,4 +99,13 @@ def pretty_print(self, text: dict, text_input):
         """
 
     def print_json(self, text: dict):
-        return json.dumps(text, indent=4)
+        if text['Regexes'] == None:
```

Comment:
Please format it all using black.🖤

Reply:
I will get it done after making the code changes.